src/App.jsx

```diff
@@ -14,45 +23,142 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+let previousGoods = [...goodsFromServer];
+
+const getSortedGoods = (goods, sortMethod, isReverse) => {
+  let newGoods = [...goods];
+  const isReverseSwitched = isReverse !== previousReverse;
```

Comment:
Better create a state for `isReversed` instead

Reply:
I have rebuilt the task with 2 states